src/tests/ftest/util/ior_test_base.py

```diff
@@ -37,6 +37,7 @@ def __init__(self, *args, **kwargs):
         self.processes = None
         self.hostfile_clients_slots = None
         self.container = None
+        self.disable_caching = True
```

Comment:
I'm not sure this works, since in dfuse_test_base.py, a new Dfuse() command is created each time it runs, and I don't think IorTestBase.disable_caching is taken into account.
https://github.com/daos-stack/daos/blob/26a478afa6252c99bfa89cee17b0414a28629a63/src/tests/ftest/util/dfuse_test_base.py#L39-L48

Reply:
You may well be right, we'll have to see.  An older version of this PR failed IOR and CI hasn't been stable enough to run this code yet.  I was hoping that object inheritance would take care of this but I'll admit to not having a firm grasp on how ftest works.  We should have results in an hour or so and we can take a close look then.

Reply:
I do think it would be ideal if the ior results work "as-is". I.e. with the default of caching enabled.

Otherwise, if we do need to disable...
The problem here is that the Dfuse command itself is not part of the test "base" inheritance chain. If you want IorTestBase.disable_caching to be taken into account automatically for all tests inheriting IorTestBase, I _**think**_ we might need something like:
```
self.dfuse = Dfuse(hosts, self.tmp)
# Handle default caching behavior
if self.disable_caching:
    self.dfuse.disable_caching = True
# Now get all params from config (.yml), including disable_caching
self.dfuse.get_params(self)
```
And to be more clear in DfuseTestBase, we should do this in `DfuseTestBase.__init__`
```
def __init__(...):
    # By default, do not disable and get the value from config
    self.disable_caching = False
```
That way IorTestBase.disable_caching is setting the inherited DfuseTestBase.disable_caching.
And when DfuseTestBase runs dfuse, it will first set DfuseTestBase.disable_caching, and then optionally override that with the value from the test config (.yml).
The end result is:
- By default, dfuse enables caching.
- By default, any test inheriting IorTestBase will disable caching
- Individual test cases can enable caching by putting this in the test config (.yml):
  ```
  dfuse:
    disable_caching: false
  ```

But yes, we can wait until the results are in to see if this is necessary.

Reply:
> I do think it would be ideal if the ior results work "as-is". I.e. with the default of caching enabled.

It would, if we only ran IOR on a single node.

> Otherwise, if we do need to disable...
> The problem here is that the Dfuse command itself is not part of the test "base" inheritance chain. If you want IorTestBase.disable_caching to be taken into account automatically for all tests inheriting IorTestBase, I _**think**_ we might need something like:
> 
> ```
> self.dfuse = Dfuse(hosts, self.tmp)
> # Handle default caching behavior
> if self.disable_caching:
>     self.dfuse.disable_caching = True
> # Now get all params from config (.yml), including disable_caching
> self.dfuse.get_params(self)
> ```

This is the bit I was missing.  What I've done is just added the option, and set it in the yaml for all tests where it's needed.  I've pushed this, and triggered a daily_regression build to be sure but the previous one only hit ior and fio failures, and I've already tested both of those since.

Reply:
That is probably the simplest approach, thought I'm not sure if other tests do or do not need to be ran.